internal/resources/metal/project/resource_test.go

```diff
@@ -340,6 +340,7 @@ func testAccMetalProjectConfig_organization(r string) string {
 	return fmt.Sprintf(`
 resource "equinix_metal_organization" "test" {
 	name = "tfacc-project-%s"
+	description = "quux"
```

Comment:
It doesn't look like description is required for organizations; why is this being added in the test?

Reply:
Was this done to address this comment? https://github.com/equinix/terraform-provider-equinix/pull/568/files#r1507920057

Reply:
If so, as mentioned in that comment, that test failure shouldn't be fixed by changing this test; it should be fixed by making the organization framework resource handle the `description` field the same way the SDKv2 resource handled it.